CLAUDE.md

```
@@ -0,0 +1,94 @@
+yarn typecheck
+
+# Serve production build locally
+yarn serve
+
+# Clear build cache
+yarn clear
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)